easybuild/easyconfigs/g/GAMESS-US/rungms.patch

```diff
@@ -0,0 +1,145 @@
+This patch adds support for the SLURM scheduler (without removing support for
+the PBS or SGE schedulers), while also setting other important values (such as
+scratch directories) and patching the rungms script so that other hardcoded
+values can be set by means of a regular expression during the EasyBuild
+configuration process.
+You may need to tweak the values for SCR and USERSCR to adjust them for your
+own site.
+
+Written by Benjamin Roberts, New Zealand eScience Infrastructure
+University of Auckland, Auckland, New Zealand
+--- gamess.orig/rungms	2013-05-18 04:15:11.000000000 +1200
++++ gamess/rungms	2015-01-29 17:39:53.359679532 +1300
+@@ -59,17 +59,23 @@
+ #       both Sun Grid Engine (SGE), and Portable Batch System (PBS).
+ #       See also a very old LoadLeveler "ll-gms" for some IBM systems.
+ #
+-set TARGET=sockets
+-set SCR=/scr/$USER
+-set USERSCR=~$USER/scr
+-set GMSPATH=/u1/mike/gamess
++set TARGET=self.connectivity
```

Comment:
you can't access Python variables in a script, and `self` is a Python object reference, so it will not even be defined outside of a Python session